commands/new_function.go

```diff
@@ -81,12 +85,27 @@ the "Dockerfile" lang type in your YAML file.
 		return fmt.Errorf("you must supply a function language with the --lang flag")
 	}
 
-	PullTemplates("")
+	existingYaml := len(yamlFile) > 0
+	var stackFileName string
+	if !existingYaml {
+		// We will create a new YAML file for this function
+		stackFileName = functionName + ".yml"
+	} else {
+		stackFileName = yamlFile
```

Comment:
Looks like this change isn't pushed through to the point at which the write is echo'd to stdout. For example creating a function `myfn` and passing the yaml explicitly `--yaml stack.yaml` results in the following being output to the user.
```
Function created in folder: myfn
Stack file written: myfn.yml
```
It should reflect the stack file written to:
```
Stack file written: stack.yml
```